src/pages/Designer/Designer.js

```diff
@@ -190,24 +176,15 @@ const Designer = () => (
           margin={{ top: 'medium' }}
           gap="large"
         >
-          <Box direction="column">
-            <Image src="Designer_4.svg" />
-            <Text alignSelf="center" color="altGrey">
-              Creating in Light and Dark modes
-            </Text>
-          </Box>
-          <Box direction="column">
-            <Image src="Designer_5.svg" />
-            <Text alignSelf="center" color="altGrey" margin="medium">
-              Importing your own theme
-            </Text>
-          </Box>
-          <Box direction="column">
-            <Image src="Designer_6.svg" />
-            <Text alignSelf="center" color="altGrey">
-              Easy publishing
-            </Text>
-          </Box>
+          <ScreenshotCard
+            src="Designer_4.svg"
+            text="Creating in Light and Dark modes"
+          />
+          <ScreenshotCard
+            src="Designer_5.svg"
+            text="Importing your own theme"
```

Comment:
The prop named `text` is confusing, could it be `label` instead? or any other suggestions?